src/Microsoft.ML.Transforms/FourierDistributionSampler.cs

```diff
@@ -68,9 +68,15 @@ internal abstract class FourierRandomNumberGeneratorBase
     }
 
     /// <summary>
-    /// The Gaussian kernel is defined as k(x,y)=exp(-gamma*|x-y|_2^2). The distribution that is the Fourier transform of
-    /// this kernel is the Normal distribution with variance 2*gamma.
+    /// The Gaussian kernel.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// The Gaussian kernel is defined as $k(x,y) = \exp(-\gamma||x-y||^2)$. The distribution that is the Fourier transform of
+    /// this kernel is the Normal distribution with variance $2 \gamma$.
```

Comment:
What does this mean?
```The distribution that is the Fourier transform of this kernel is the Normal distribution with variance $2 \gamma$.```

Reply:
This refers to the comment on the base class: that the Fourier transform of a continuous positive definite kernel is a non-negative measure. 

Perhaps we should mention this in the documentation of each implementation? Alternatively, we can remove the sentences mentioning the distributions, since the methods on these classes that sample from the distributions are internal anyway.

What do you guys think?

---
In reply to: [277855942](https://github.com/dotnet/machinelearning/pull/3545#discussion_r277855942) [](ancestors = 277855942)

Reply:
it's not clear to me either. please reword.

---
In reply to: [277889125](https://github.com/dotnet/machinelearning/pull/3545#discussion_r277889125) [](ancestors = 277889125,277855942)